custom/icds_reports/reports/awc_reports.py

```diff
@@ -1114,7 +1114,7 @@ def get_awc_report_pregnant(order, reversed_order, awc_id):
         'case_id', 'person_name', 'age_in_months', 'opened_on', 'edd', 'trimester', 'anemic_severe',
         'anemic_moderate', 'anemic_normal', 'anemic_unknown', 'num_anc_complete', 'pregnant',
         'num_rations_distributed', 'last_date_thr'
-    )
+    )[:10]
```

Comment:
I dont think this is correct, because if there are less than 10 months in whcih a case was active it will still scan all data ever. I think you want to add a filter like `month > current_month - 10 months`, because we care about time passed not number of records